src/app/pages/send/send-crypto-asset-form/form/stacks-sip10/use-sip10-send-form.tsx

```diff
@@ -22,23 +21,19 @@ import {
   useGenerateFtTokenTransferUnsignedTx,
 } from '@app/store/transactions/token-transfer.hooks';
 
-import { useStacksFtRouteState } from '../../family/stacks/hooks/use-stacks-ft-params';
 import { useSendFormNavigate } from '../../hooks/use-send-form-navigate';
 import { useStacksCommonSendForm } from '../stacks/use-stacks-common-send-form';
 
 export function useSip10SendForm() {
-  const { symbol } = useParams();
-  const { contractId: routeContractId } = useStacksFtRouteState();
-  const [contractId, setContractId] = useState('');
-
-  const generateTx = useGenerateFtTokenTransferUnsignedTx(contractId);
+  const { symbol, contractId = '' } = useParams();
```

Comment:
Rather than setting as `''` maybe we could:

```ts
if (!contractId) {
  toast.error('Unable to find contract id);
  return <Navigate path={} />
}
```

Imo this is preferable as we avoid a type safe, but illegal, value. Or does this cause type issues elsewher?